### PR TITLE
Introduce staging configuration

### PR DESCRIPTION
**Pull request by mattThousand**
### Why
We'd like to suppress the debug menu for release builds, and also ensure that builds are not pointed to the release environment during testing

### This Commit
This commit introduces a `STAGING` env configuration that suppresses the debug menu and points to the dev server, as well as an `Adhoc` scheme for `iOS` that uses the `.env.bt` configuration

### CODE REVIEW

**Comment by juancruzgs** [comment]
@mattThousand According to this [card](https://trello.com/c/vFqRhwLM/417-as-a-dev-team-when-we-make-a-release-build-end-users-do-not-see-debug-info-menus-etc) we shouldn't show the debug menu on staging builds , right?

File: android/app/build.gradle [hunk 1]
```diff
@@ -6,6 +6,7 @@ apply plugin: 'realm-android'
 
 project.ext.envConfigFiles = [
     debug: ".env.bt",
+    staging: ".env.bt.staging",
```
Comment (juancruzgs) [inline android/app/build.gradle:9]: This won't compile I think, more changes are needed to add a new build variant
I can make those changes here

Comment (mattThousand) [reply android/app/build.gradle:9]: @juancruzgs thanks!

**Comment by mattThousand** [comment]
> @mattThousand According to this [card](https://trello.com/c/vFqRhwLM/417-as-a-dev-team-when-we-make-a-release-build-end-users-do-not-see-debug-info-menus-etc) we shouldn't show the debug menu on staging builds , right?
> If it does show the menu, what's the difference between DEV and STAGING?

@juancruzgs thanks, you are right

**Pull request merged**